### DISCUSSION TITLE
Is it possible to ungroup the device tiles? Eg split the purifier and sensor tiles

### DISCUSSION THREAD

**Discussion by pankit1234** [RaresAil/homebridge-levoit-air-purifier#33] [Ideas]
Wondering if its possible to ungroup the purifier from the sensor? Some devices and plugins give you the option to ‘Group with other accessories’ or ‘show as single tile’ / ‘Show as separate tiles’. 

Currently to reach the settings for the purifier via the Home App you have to do the following

- Long Press device icon
- Scroll down or hit gear icon
- Select accessories
- Long Press the Purifier icon
- Scroll down or hit gear icon
- Change your settings

if the tiles were separate this would be reduced to 

- Long Press Purifier Icon
- Scroll down or hit gear icon
- Change your settings

Would this be possible?

**Comment by RaresAil** [comment]
I will try to work on this

**Comment by RaresAil** [comment]
Hey @pankit1234 This feature was implemented in `v1.2.0`
